apidef/api_definitions.go

```diff
@@ -597,6 +597,7 @@ type UptimeTests struct {
 }
 
 type AuthConfig struct {
+	Name              string          `mapstructure:"name" bson:"name" json:"name"`
```

Comment:
should we name it `AuthConfigName` or just `Name`? it's named `AuthConfigName` in the contract